themes/src/main/resources/theme/base/admin/resources/js/controllers/realm.js

```diff
@@ -3218,20 +3223,35 @@ module.controller('ClientPoliciesProfilesEditExecutorCtrl', function($scope, rea
                     break;
                 }
             }
+
+            for (var j=0 ; j < $scope.executorType.properties.length ; j++) {
```

Comment:
I am pretty sure this will have some conflicts with https://github.com/keycloak/keycloak/pull/8093

Reply:
The UI fixes were moved directly to #8093.